### PR TITLE
Release notes update for 5-19-15

### PR DESCRIPTION
**Pull request by mhoeber**

### CODE REVIEW

**Comment by mhoeber** [comment]
@catong @benpatterson @explorerleslie @cptvitamin 

added release info for LMS and documentation

File: en_us/release_notes/source/2015/index.rst [hunk 1]
```diff
@@ -2,6 +2,8 @@
 .. toctree::
    :maxdepth: 1
    
+   05-19-2015
+   05-12-2015
```
Comment (catong) [inline en_us/release_notes/source/2015/index.rst:6]: Did 5-12-2015 get added twice?

File: en_us/release_notes/source/2015/lms/lms_0519_2015.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
+=================================
+Weighted Problem Grade Report
+=================================
+
+For any course, you can now calculate grades for problems and generate a report
+that can be downloaded. For more information, see DOC LINK. (TNL-1951)
```
Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0519_2015.rst]: that can be downloaded from the instructor dashboard. This new report includes, for each graded problem, a learner's earned and possible points, and their total score, expressed as a decimal.
Doc link: edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/running_course/course_grades.html#problem-report

Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0519_2015.rst]: @mhoeber Also need to remove the associated blurb from the "Coming Soon" file.

Comment (explorerleslie) [inline en_us/release_notes/source/2015/lms/lms_0519_2015.rst]: @catong did we decide to not call it the "weighted" problem grade report? cc @mhoeber

**Comment by explorerleslie** [comment]
@mhoeber one comment, otherwise, :+1: The release just shipped so you may want to update soon....

File: en_us/release_notes/source/2015/lms/lms_0519_2015.rst (hunk 2 above)
Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0519_2015.rst]: Oops, I missed the heading. Yes, that was an internal name that changed. It is the "Problem Grade Report"

File: en_us/release_notes/source/2015/documentation/doc_0519_2015.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+`Change Log`_.
+
+==================================
+edX Research Guide
+==================================
+
+This guide includes updated descriptions of the ``pause_video``,
+``play_video``, and ``stop_video`` `video interaction events`_ to include the
+effects of a **Video Start Time** or **Video Stop Time**.
+
```
Comment (catong) [inline en_us/release_notes/source/2015/documentation/doc_0519_2015.rst:22]: Two events have been added to the instructor events section for generating and downloading reports from the instructor dashboard. http://devdata.readthedocs.org/en/latest/internal_data_formats/tracking_logs.html#instructor-events

File: en_us/release_notes/source/2015/lms/lms_0519_2015.rst (hunk 2 above)
Comment (catong) [inline en_us/release_notes/source/2015/lms/lms_0519_2015.rst]: @mhoeber Possibly an impact to the Analytics files too: 2 new events have also been added: 
edx.instructor.report.downloaded and edx.instructor.report.requested.
Instructor Events section: http://devdata.readthedocs.org/en/latest/internal_data_formats/tracking_logs.html#instructor-events

File: en_us/release_notes/source/2015/lms/lms_0519_2015.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Problem Grade Report for Enrolled Students`_. (TNL-1951)
+
+=================================
+Accessibility Improvements
+=================================
+
+This release includes accessibility improvements for learners who use a
+keyboard or screen reader to access the LMS.
+
+* The **Courseware** tab now includes role and aria-label attributes.
```
Comment (cptvitamin) [inline en_us/release_notes/source/2015/lms/lms_0519_2015.rst:18]: Should be "The LMS now has a region with a role of "main" and a descriptive aria-label allowing users to quickly navigate to the main content area using landmarks."

**Pull request merged**